sql/pokedex_hb.sql

```diff
@@ -40,8 +40,12 @@ DROP TABLE IF EXISTS `status`;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `status` (
```

Comment:
Any reason there's no id column to refer to the identity of each entry in this table?

Reply:
No id necessary for linking tables.